src/Mod/Path/PathCommands.py

```diff
@@ -43,6 +51,127 @@ def translate(ctxt, txt):
 __author__ = "sliptonic"
 __url__ = "https://www.freecadweb.org"
 
+class _CommandCamoticsSimulate:
+
+    def __init__(self):
+        self.obj = None
+        self.sub = []
+        self.active = False
+
+        self.tooltemplate = {
+            "units": "metric",
+            "shape": "cylindrical",
+            "length": 10,
+            "diameter": 3.125,
+            "description": ""
+        }
+
+        self.workpiecetemplate = {
+            "automatic": "false",
+            "margin": 0,
+            "bounds": {
+                "min": [0, 0, 0],
+                "max": [0, 0, 0]}
+        }
+
+        self.camoticstemplate = {
+            "units": "metric",
+            "resolution-mode": "medium",
+            "resolution": 1,
+            "tools": {},
+            "workpiece": {},
+            "files": []
+        }
+
+    def GetResources(self):
+        return {'Pixmap': 'Path_Camotics',
+                'MenuText': QtCore.QT_TRANSLATE_NOOP("Path_Camotics", "Camotics"),
+                'Accel': "P, C",
+                'ToolTip': QtCore.QT_TRANSLATE_NOOP("PatCamoticsop", "Simulate using Camotics"),
+                'CmdType': "ForEdit"}
+
+    def IsActive(self):
+        if bool(FreeCADGui.Selection.getSelection()) is False:
+            return False
+        try:
+            job = FreeCADGui.Selection.getSelectionEx()[0].Object
+            return isinstance(job.Proxy, PathScripts.PathJob.ObjectJob)
```

Comment:
no simulation of ops?

Reply:
Doesn't really make sense.  We're building a camotics project which takes a stock, tools, and one or more NC files.  To get real gcode, we have to postprocess.  To do simulation of only one op means disabling the others, postprocessing, and re-enabling.   Maybe we should look at that for v2.